substrate-node/runtime/src/constants.rs

```diff
@@ -86,27 +86,29 @@ pub mod fee {
 
 #[cfg(test)]
 mod tests {
-    use super::currency::{CENTS, DOLLARS, MILLICENTS};
+    use super::currency::{CENTS, MILLICENTS};
     use super::fee::WeightToFeeStruct;
     use crate::MaximumBlockWeight;
     use frame_support::weights::constants::ExtrinsicBaseWeight;
     use frame_support::weights::WeightToFee;
-
+    
     #[test]
     // This function tests that the fee for `MaximumBlockWeight` of weight is correct
     fn full_block_fee_is_correct() {
         // A full block should cost 16 DOLLARS
-        println!("Base: {}", ExtrinsicBaseWeight::get());
+        log::info!("MaxBlockWeight: {:?}", MaximumBlockWeight::get());
+
         let x = WeightToFeeStruct::weight_to_fee(&MaximumBlockWeight::get());
-        let y = 16 * DOLLARS;
+        // I think this should be 23.31120 dollar or 2331120 millicents
+        let y = 2331120 * MILLICENTS;
```

Comment:
I believe the test was wrong: 
FEE_BLOCK = MAX_AMT_EXTRINSICS_PER_BLOCK * FEE_PER_EXTRINSIC
 => MAX_AMT_EXTRINSICS_PER_BLOCK = MAX_WEIGHT_PER_BLOCK/WEIGHT_PER_EXTRINSIC
 => MAX_AMT_EXTRINSICS_PER_BLOCK ≃ 23311.38
 => FEE_PER_EXTRINSIC = 1/10 cents
FEE_BLOCK ≃ 2331.138 cents = 23.31138 dollar